python/cinn/compiler/compiler.py

```diff
@@ -35,4 +39,3 @@ def compile(fn, just_convert=False, jit_inputs_signature=[], **kwargs):
 
     if just_convert:
         return llir_func
-    return llir_func
```

Comment:
Maybe problem here?

Reply:
The next pr will add runtime, and what will be returned here is the CINN runtime module encapsulation. For now, just convert python dsl to cinn c++ ir as normal.